memgpt/local_llm/ollama/api.py

```diff
@@ -0,0 +1,62 @@
+import os
+from urllib.parse import urljoin
+import requests
+import tiktoken
+
+from .settings import SIMPLE
+from ...constants import LLM_MAX_TOKENS
+from ...errors import LocalLLMError
+
+HOST = os.getenv("OPENAI_API_BASE")
+HOST_TYPE = os.getenv("BACKEND_TYPE")  # default None == ChatCompletion
+MODEL_NAME = os.getenv("OLLAMA_MODEL")  # ollama API requires this in the request
+OLLAMA_API_SUFFIX = "/api/generate"
+DEBUG = False
+
+
+def count_tokens(s: str, model: str = "gpt-4") -> int:
```

Comment:
Should move to utils, also appears in webui/api.py

Reply:
Refactored into utils.py